routemaster/state_machine/utils.py

```diff
@@ -186,3 +186,43 @@ def context_for_label(
         feeds,
         accessed_variables,
     )
+
+
+def process_transitions(app: App, label: LabelRef):
```

Comment:
Type signature should have ` -> None` since there's no return value. (The current signature will be interpreted as ` -> Any`.)

Reply:
33e4e115eb76740d60e3fc1c2b618d6e3237d887